CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 ## Dependencies Needed
 
 - CMake (tested with 3.23)
-- Boost (tested with 1.71.0)
+- Boost (tested with 1.73.0/1.79.0)

```

Comment:
Makes me wonder if we should just update Boost to 1.79 for all platforms... also does this work when pulled into our pipelines _inside_ NI?

Reply:
NI build is using Boost 1.71.0 and I've verified that the build is still working with all the changes we made thus far.